mobile/src/bundles/common/components/searchable-dropdown/searchable-dropdown.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import {
+    type Control,
+    type FieldPath,
+    type FieldValues,
+} from 'react-hook-form';
+import { TextInput } from 'react-native';
+
+import {
+    ScrollView,
+    Text,
+    TouchableOpacity,
+    View,
+} from '~/bundles/common/components/components';
+import { TextCategory } from '~/bundles/common/enums/enums';
+import { useFormController } from '~/bundles/common/hooks/hooks';
+import { globalStyles } from '~/bundles/common/styles/styles';
+
+import { styles } from './styles';
+
+type Properties<T extends FieldValues> = {
+    control: Control<T, null>;
+    name: FieldPath<T>;
+    hasError?: boolean;
+    items: string[];
+    placeholder?: string;
+    onItemSelect: (item: string) => void;
+};
+
+const SearchableDropdown = <T extends FieldValues>({
```

Comment:
Let's add tags to this shared component as well since it's related on this selector

Reply:
So this will be multi select by default

Reply:
We should also filter from dropdown already selected value

Reply:
Let's create for auto-complete separate AutocompleteMuiltiSelector and AutocompleteSelector, then it should work well, AutocompleteMuiltiSelector we'll use for hard skills and Preferred languages, AutocompleteSelector we'll use for Current location.
AutocompleteMuiltiSelector will have tags inside of it, and value for this selector should always be []!! 
AutocompleteSelector will work as common selector but with ability to search values